docs/backend/content-types/index.md

```diff
@@ -12,15 +12,223 @@ myst:
+  <action action_id="edit"
+          category="object"
+          condition_expr="not:object/@@plone_lock_info/is_locked_for_current_user|python:True"
+          icon_expr="string:toolbar-action/edit"
+          title="Edit"
+          url_expr="string:${object_url}/edit"
+          visible="True"
+          i18n:attributes="title"
+  >
+    <permission value="Modify portal content" />
+  </action>
+
+</object>
+```
+
+The `name` attribute on the root element in the XML must match the name in the filename and the name listed in `types.xml`.
+
+Set the `i18n:domain` to the i18n domain which includes translations for this content type. This is usually the same as the name of the Python package which contains the content type.
+
+The XML sets a number of FTI properties:
+
+`action` elements
+:   Defines additional {doc}`actions <portal-actions>` which are available for this content type.
+
+`add_permission`
+:   Id of the permission controlling whether the current user has permission to add this content type.
+
+`allow_discussion`
+:   Boolean.
+    Controls whether Plone's commenting system is enabled for this content type.
+
+`allowed_content_types`
+:   List of content types which can be added inside this one.
+    Only used if `filter_content_types` is True.
+
+`behaviors`
+:   List of {doc}`behaviors <behaviors>` enabled for this content type.
+
+`description`
+:   Short description displayed in the UI.
+
+`factory`
+:   Name of the factory adapter used to create new instances of the content type.
+    Usually the same as the content type name.
+
+`filter_content_types`
+:   Boolean.
+    Controls which content types can be added inside this one.
+    If `True`, allow only the types listed in `allowed_content_types`.
+    If `False`, allow any content type that the user has permission to add.
+
+`global_allow`
+:   Boolean.
+    Set to `True` to allow adding the content type anywhere in the site where the user has permission.
+    Set to `False` to only allow adding it inside other content types that include this one in `allowed_content_types`.
+
+`klass`
+:   Dotted path to the Python class for this content type.
+
+`model_file`
+:   Location of an XML file to load as the content type's schema.
+    This is an alternative to `schema` and `model_source`.
+
+`model_source`
+:   Inline XML schema for the content type.
+    This is an alternative to `schema` and `model_file`.
+
+`schema`
+:   Dotted path to the schema for this content type.
+
+`title`
+:   The name of the content type displayed in the UI.
+
+
+The following FTI properties are only used in the Classic UI:
+
+`add_view_expr`
+:   TALES expression returning the URL for the form to add a new item of this content type.
+
+`alias` elements
+:   Controls a mapping from URL to views.
+    It's not common to change this.
+
+`default_view`
+:   Name of the default view used to display this content type.
+
+`default_view_fallback`
+:   ```{todo} add definition of `default_view_fallback `.
```

Comment:
It looks like [MyST still chokes on the `todo` directive as the first line in a definition](https://deploy-preview-1498--6-docs-plone-org.netlify.app/backend/content-types/index.html).

![Screen Shot 2023-05-15 at 10 14 02 AM](https://github.com/plone/documentation/assets/102112/576ed3be-def3-4386-b8e0-d32bd83dced2)

Let's keep it simple and use plain old text, and remove the directives and backtick fences.

And I'm clearly lacking ☕ . With any TODO, it is a good idea to create an issue and reference the issue in the TODO to encourage and facilitate contributions.

```suggestion
:   TODO: add definition of `default_view_fallback `.
    See [link to issue].
```

Reply:
@stevepiercy I did some more research, added a definition for default_view_fallback, and removed link_target because it appears to be unused and not required.